services/worker/src/worker/dtos.py

```diff
@@ -333,7 +333,7 @@ class DatasetCompatibleLibrariesResponse(TypedDict):
     formats: list[DatasetFormat]
 
 
-DatasetModality = Literal["image", "audio", "text", "video", "geospatial", "3d"]
+DatasetModality = Literal["image", "audio", "text", "video", "geospatial", "3d", "tabular", "timeseries"]
```

Comment:
maybe name it "time-series" ?

Reply:
we use `timeseries` in moon-landing, so... better to stick with this, unless it's really an issue for you :P